apps/docs/LICENSE.md

```diff
@@ -1 +1,7 @@
-See [tldraw license](https://github.com/tldraw/tldraw/blob/master/LICENSE.md)
+Copyright 2024 tldraw Inc.
```

Comment:
it'd be good to write MIT License at the top. it makes it clearer to the person reading this.

Reply:
same as the other packages below!